packages/flutter/test/widget/overflow_box_test.dart

```diff
@@ -32,4 +32,20 @@ void main() {
       expect(box.size, equals(const Size(100.0, 50.0)));
     });
   });
+
+  test('OverflowBox implements debugFillDescription', () {
+    var description = [];
```

Comment:
And, when https://github.com/flutter/flutter/issues/1220 lands, we'll be able to write a rule for this, too :)

cc @pq